lib/types/types.d.ts

```diff
@@ -89,7 +89,13 @@ export type UserRemovedData = {
     channel_id?: string;
     remover_id: string;
 } & BaseData;
-export type WebsocketEventData = EmptyData | HelloData | CallStartData | UserDisconnectedData | UserConnectedData | UserMutedUnmutedData | UserVoiceOnOffData | UserScreenOnOffData | UserRaiseUnraiseHandData | EmojiData | UserReactionData | CallHostChangedData | CallRecordingStateData | UserState | UserDismissedNotification | CallStateData | JobStopData | UserRemovedData;
+export type LiveCaptionData = {
+    channelID: string;
+    user_id: string;
+    session_id: string;
+    text: string;
```

Comment:
Will we ever need a language field or is it something we should describe at a higher level?

Reply:
I've been thinking about it, and afaik I don't think so. Unless we eventually have a translation function as well, which I can't see anytime soon.